lib/linguist/languages.yml

```diff
@@ -1182,6 +1182,15 @@ GAS:
   tm_scope: source.assembly
   ace_mode: assembly_x86
 
+GCC machine description:
```

Comment:
Should we title case the name? (We can't change it after; so whatever we do, we better be sure about it :stuck_out_tongue: )